python/lsst/dax/imgserv/vo/imageSODA.py

```diff
@@ -79,10 +82,13 @@ def do_async(self, params: dict) -> object:
 
         Returns
         -------
-        xml: `str`
+        resp: `str`
+            the status.
```

Comment:
Doesn't it return the URL, not the status?

Reply:
it's now returning the task_id to track the newly created. job.